packages/sdk/src/indexer/schemas.ts

```diff
@@ -253,7 +253,12 @@ export const IndexerAPICommentReferenceURLWebPageSchema = z.object({
     .object({
       title: z.string(),
       description: z.string().nullable(),
-      image: z.string().url(),
+      image: z.union([
+        z.string().url(),
+        z
+          .string()
+          .regex(/^\/[^/\s]+$/, "Must be a valid file path starting with /"),
```

Comment:
I don't think this is correct solution. From open graph spec the open graph url must be a full URL.

If we want to support the relative paths we should resolve them on indexer side and return the full URL or ignore them as they aren't valid and return null.